src/components/Badge/__stories__/Badge.stories.js

```diff
@@ -0,0 +1,99 @@
+import Badge from "../Badge";
+import person from "./assets/person.png";
+import { ExternalPage, WhatsNew } from "../../Icon/Icons";
+import Link from "../../Link/Link";
+import Button from "../../Button/Button";
+import Flex from "../../Flex/Flex";
+import Avatar from "../../Avatar/Avatar";
+import { createStoryMetaSettingsDecorator } from "../../../storybook";
+import { createComponentTemplate, StoryDescription } from "vibe-storybook-components";
+
+const metaSettings = createStoryMetaSettingsDecorator({
+  component: Badge,
+  enumPropNamesArray: ["anchor", "alignment", "type"],
+  iconPropNamesArray: [],
+  actionPropsArray: [],
```

Comment:
Same comment about this decorator, I think 100% of our actions start with `on*`, I'm pretty sure we can set it globally to identify these props as actions and log them in the addon

Reply:
Tried to do it once btw and found smth strange about it...
Not sure in which story it was (maybe in Dropdown or smth) and which actions it was, but eventually I manually added a few new actions to the array and it somehow broke functionality of the component itself... I was very surprised and gave up this idea for a while...